scripts/transfer_approve_deposit.js

```diff
@@ -18,8 +20,8 @@ module.exports = async (callback) => {
     const GnosisSafe = artifacts.require("GnosisSafe")
     const masterSafe = await GnosisSafe.at(argv.masterSafe)
 
-    const deposits = require(argv.depositFile)
-    // TODO - make a simpler to construct deposit file style
+    const deposits = JSON.parse(await fs.readFile(argv.depositFile, "utf8"))
+    // TODO - make it simpler to construct deposit file style
```

Comment:
I think this TODO should be removed if we aren't going to TODO it.

Reply:
Fair enough, I don't see this happening any time soon.